pcsx2/des.h

```diff
@@ -0,0 +1,65 @@
+ /**
```

Comment:
Might want to throw that in 3rdparty instead of just dropping it in the main folder